src/js/components/Button/StyledButton.js

```diff
@@ -248,6 +248,7 @@ const StyledButton = styled.button.withConfig(styledComponentsConfig)`
     props.pad === true &&
     props.hasIcon &&
     !props.hasLabel &&
+    // TO DO theme object ? theme for padding?
     `
     padding: ${props.theme.global.edgeSize.small};
```

Comment:
`theme.button.icon.pad` ???

Reply:
It looks like this padding only gets applied when there isn't a label. Maybe the name should reflect that this is for an icon only button?
`theme.button.iconOnly.pad`?

Reply:
yes `iconOnly` is good!